features/main/exception_to_status.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Using exception_to_status config
+  As an API developer
+  I can customize the status code returned if the application throw an exception
+
+  @createSchema
+  Scenario: Configure status code via the operation exceptionToStatus to map custom NotFound error to 404
+    When I add "Content-Type" header equal to "application/ld+json"
+    And I send a "GET" request to "/dummy_exception_to_statuses/123"
+    Then the response status code should be 404
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
+
+  @createSchema
```

Comment:
Maybe we can remove some `createSchema` since there are not really useful and time consuming.
I known tests will not be independently anymore, but we already do it everywhere (IMO we should use transactions to speed up tests instead).